app/server/db/requests.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+import PouchDB from 'pouchdb';
+
+export default class DBConnection {
+	constructor() {
+		this.localDB = new PouchDB('agile');
+		this.remoteDB = new PouchDB('http://localhost:5984/agile');
+
+		this.localDB.sync(this.remoteDB, {
+		  	live: true
+		}).on('complete', function () {
+		  	console.log('dbs replicated')
+		}).on('error', function (err) {
+			console.log('some troubles while replicating', err);
+		});;
+
+		this.remoteDB.info().then(function (info) {
+		  console.log('!!!!!!!!!!!!', info);
+		});
+
+		this.localDB.info().then(function (info) {
+		  console.log('@@@@@@@@@@@@@@@', info);
+		})
+	}
+
+	addTask(task) {
+		task._id = task.name.toLowerCase().replace(/\s/g, '');
+		this.remoteDB.put(task).then(function (response) {
```

Comment:
return this.remoteDB.put(task) better i don't think that resolver in this method a good idea